Docs/Api/addEventSample.md

```diff
@@ -0,0 +1,28 @@
+### Example curl command to add an event from a JSON file in the same directory
```

Comment:
Building some docs, awesome!

These probably need fleshing out in the future but this will do nicely for now.